README.md

```diff
@@ -30,6 +30,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     A Brazilian initiative to provide FREE classes on javascript to women in a male-free environment.
 
+  - [JsLadies BR](https://github.com/jsladiesbr) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    A group for join up women interested in web development and javascript with open and free meetings.
```

Comment:
* 'join up women' is that correct?

Other than that it looks fine, do you think you can research a little more and add a line or two on this if possible? At the moment there is not too much detail. If not possible it's fine, and I'll merge it.

Reply:
Hello @harsh183 :)
I'm not sure about the "join up". Do you think "A group to unite women" gets better?

Reply:
I think that's a tad too descriptive. Maybe just go with plain and simple, 'A group for women interested in ...'